.github/workflows/ci.yml

```diff
@@ -55,6 +55,14 @@ jobs:
           dune-cache: true
           opam-disable-sandboxing: true
 
+      - name: Install xxhash (ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get install -y libxxhash-dev
```

Comment:
would it mean that all consumers of server-reason-react need to have this installed?

Reply:
Yes :( Alternatives can be:
- put the melange.ppx in a separate package
- reimplement the xxhash lib in OCaml
- ...

Reply:
and what about adding libxxhash-dev depext?